Eevee.Sleep.Bot/Controllers/Mongo/InGameAnnouncement/InGameAnnoucementHistoryControllers.cs

```diff
@@ -0,0 +1,40 @@
+using Eevee.Sleep.Bot.Models.InGameAnnouncement;
+using MongoDB.Driver;
+
+namespace Eevee.Sleep.Bot.Controllers.Mongo.InGameAnnouncement;
+
+public static class InGameAnnouncememntHistoryController {
+    public static async Task Upsert(InGameAnnouncementDetailModel model) {
+        using var session = await MongoConst.Client.StartSessionAsync();
+        session.StartTransaction();
+
+        try{
+            await MongoConst.InGameAnnouncementHistoryCollection.InsertOneAsync(model);
+            var count = await MongoConst.InGameAnnouncementHistoryCollection
+                .CountDocumentsAsync(Builders<InGameAnnouncementDetailModel>.Filter.Where(x => x.AnnouncementId == model.AnnouncementId));
+
+            if (count > 3)
+            {
```

Comment:
Put `{` to the same line, so `if (count > 3) {`

Also make `3` a `const` from a class specifically for in-game announcements somewhere.